docs/structured-streaming-programming-guide.md

```diff
@@ -2385,6 +2385,11 @@ Here are the configs regarding to RocksDB instance of the state store provider:
     <td>Total memory to be occupied by blocks in high priority pool as a fraction of memory allocated across all RocksDB instances on a single node using maxMemoryUsageMB.</td>
     <td>0.1</td>
   </tr>
+  <tr>
+    <td>spark.sql.streaming.stateStore.rocksdb.allowFAllocate</td>
```

Comment:
is it just me that feels that "FAllocate" feels odd to me, why not "allowFallocate?

Reply:
Thank you @siying !  Is it the capitalization that you are finding bothersome?  I carried though the `allowFAllocate` casing from the RocksDB DBOptions class: https://javadoc.io/doc/org.rocksdb/rocksdbjni/6.8.1/org/rocksdb/DBOptions.html#allowFAllocate()

I'm definitely not tied to it, but do like maintaining consistency across the two packages.

Reply:
Keeping it consistent with RocksDB conf name sounds good to me. All good for me then.

Reply:
Nice, we can move forward. :)